__tests__/app.test.js

```diff
@@ -394,4 +395,34 @@ describe("app", () => {
         });
     });
   });
+  describe("GET: /api/users", () => {
+    test("returns an array of user objects", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body)).toBe(true);
+        });
+    });
+    test("each user object contains the correct properties", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          body.forEach((user) => {
```

Comment:
I know you do this in the following test but whenever you use a forEach loop in your tests it should be paired with something checking the length, so you don't get a false positive for this test if it were passed an empty array